components/camel-kafka/src/main/java/org/apache/camel/component/kafka/KafkaConstants.java

```diff
@@ -61,7 +61,7 @@ public final class KafkaConstants {
 
     public static final String KAFKA_DEFAULT_SERIALIZER = "org.apache.kafka.common.serialization.StringSerializer";
     public static final String KAFKA_DEFAULT_DESERIALIZER = "org.apache.kafka.common.serialization.StringDeserializer";
-    public static final String KAFKA_DEFAULT_PARTITIONER = "org.apache.kafka.clients.producer.internals.DefaultPartitioner";
+    public static final String KAFKA_DEFAULT_PARTITIONER = "";
```

Comment:
About the change itself, I would rather prefer to have it removed instead of keeping a static field with an empty String as value, but we can wait for other points of views

Reply:
+1 to remove it. 

Just, please, beware that there are a couple of tests referring to this constant. They will need to be adjusted accordingly.

Reply:
I have removed the constant